crates/ol/chain-types/src/log_payloads.rs

```diff
@@ -0,0 +1,206 @@
+//! Log payload types for orchestration layer logs.
+
+use strata_acct_types::VarVec;
+use strata_codec::{Codec, CodecError, Decoder, Encoder};
+use strata_codec_derive::Codec;
+
+/// Payload for a simple withdrawal intent log.
+///
+/// This log is emitted when a withdrawal intent is created through the bridge.
+/// It contains the withdrawal amount and destination information.
+#[derive(Debug, Clone, PartialEq, Eq, Codec)]
+pub struct SimpleWithdrawalIntentLogData {
```

Comment:
Don't really like this `Simple` prefix. How much simple? Adjective prefixed names are not generally descriptive of what they represent.

Do you really need to prefix this?